pkg/latte/node_verified_authenticator.go

```diff
@@ -56,7 +56,18 @@ func (*NodeVerifiedAuthenticator) ReactTo(ctx context.Context, deps *workflow.De
 }
 
 func (n *NodeVerifiedAuthenticator) OutputData(ctx context.Context, deps *workflow.Dependencies, workflows workflow.Workflows) (interface{}, error) {
-	return map[string]interface{}{}, nil
+	pwdAuthenticators, err := deps.Authenticators.List(ctx, n.Authenticator.UserID, authenticator.KeepType(model.AuthenticatorTypePassword))
+	if err != nil {
+		return nil, err
+	}
+
+	type NodeVerifiedAuthenticatorOutput struct {
+		PasswordEnabled bool `json:"password_enabled"`
+	}
+
+	return NodeVerifiedAuthenticatorOutput{
+		PasswordEnabled: len(pwdAuthenticators) > 0,
+	}, nil
 }
```

Comment:
Currently, `inputSelectAuthenticatorType` is accepted in `IntentLogin`, thus the data should also be outputted in that intent.
I suggest we add a field `"authenticator_types": ["password", "oob_otp_email"]` which tells the frontend which options are available.

Reply:
Updated to return authenticator types